.github/workflows/test_on_push_bricks-and-hearts.yml

```diff
@@ -50,4 +50,4 @@ jobs:
           TestDBConnectionString: 'Server=localhost,1433;Database=BricksAndHeartsTest;User Id=SA;Password=JdMsKZPBBA8kVFXVrj8d;'
 
       - name: Test with dotnet
-        run: dotnet test
+        run: dotnet test --no-build -l "console;verbosity=detailed"
```

Comment:
As you did a pull for your branch, you have merged your local changes into the remote ones, meaning this has come back, could you remove it pls :)

Reply:
yeah i just realised! Whoops...